### ISSUE TITLE
[Copied from #2541]: High Contrast: Link control missing any indication it can be click in high contrast when rendered as a button

### ISSUE THREAD

**Issue by ajinkya599** [OfficeDev/office-ui-fabric-react#5371]
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
No
 
Requested priority: (Blocking, High, Normal, Low) 
Normal

Products/sites affected: (if applicable)
VSTS Release pages
 
#### Describe the issue:

Copied from [issue #2541](https://github.com/OfficeDev/office-ui-fabric-react/issues/2541)
The issue is closed, but it seems its not fixed

The Link control will render as a button if there is no href provided. In this rendering state if the user is in high contrast white or black the text is rendered normally without any indication that it is selectable.

#### Actual behavior:
 
Text renders without hint to user that it is a click target.
<img width="891" alt="link-in-high-contrast" src="https://user-images.githubusercontent.com/11447401/42018459-c78a3846-7acf-11e8-9edf-3b9f13a1a59f.PNG">
<img width="888" alt="link-in-high-contrast-on-hover" src="https://user-images.githubusercontent.com/11447401/42018461-cc357b6c-7acf-11e8-97d6-aeb231e1308b.PNG">

 
#### Expected behavior:
 
button should look like a link

#### Repro:
 
Turn on High contrast white.
navigate to: https://developer.microsoft.com/en-us/fabric#/components/link
Look at the link text
Notice that the 2'nd link does not have visual indication that it is selectable.

**Comment by lynamemi** [comment]
One challenge here is that I don't think the Hyperlink system color is available in css. @betrue-final-final if we can't get that system color, what would be the next best thing?

**Comment by betrue-final-final** [comment]
Normally it's the yellow color in hc black

**Comment by lynamemi** [comment]
I just chatted with @betrue-final-final , and the best option we came up with is hardcoding the yellow and blue hex values for high contrast white and high contrast black. 

I'll assign this to myself and work on this implementation, but we're open to better ideas.
